core/src/scopypreferencespage.cpp

```diff
@@ -138,7 +140,7 @@ QWidget* ScopyPreferencesPage::buildGeneralPreferencesPage()
 	lay->addWidget(PreferencesHelper::addPreferenceCheckBox(p,"general_use_opengl", "Enable OpenGL plotting", this));
 	lay->addWidget(PreferencesHelper::addPreferenceCheckBox(p,"general_use_animations", "Enable menu animations", this));
 	lay->addWidget(PreferencesHelper::addPreferenceCombo(p,"general_theme", "Theme", {"default","light"}, this));
-	lay->addWidget(PreferencesHelper::addPreferenceCombo(p,"general_language", "Language",{"english","romanian"}, this));
+	lay->addWidget(PreferencesHelper::addPreferenceCombo(p,"general_language", "Language", scopy::TranslationsRepository::GetInstance()->getLanguages(), this));
```

Comment:
Can you do TranslationsRepository *t = TranslationsRepository::GetInstance(); and then connect t->getLangauges() ? it makes it more readable ..